.readthedocs.yaml

```diff
@@ -15,3 +15,8 @@ build:
     - 'git fetch --unshallow --tags || true'
     pre_install:
     - git update-index --assume-unchanged docs/conf.py
+
+# https://about.readthedocs.com/blog/2024/12/deprecate-config-files-without-sphinx-or-mkdocs-config/
```

Comment:
I guess I wouldn't put this link in the file itself, because the deprecation is temporary news and this isn't going to be adding anything if we're looking back at this file in, say, 5 years... but it's not a big deal.

Reply:
Yep, no worries, I've removed it.